app/src/main/java/com/example/easyupipayment/MainActivity.java

```diff
@@ -27,64 +31,122 @@ protected void onCreate(Bundle savedInstanceState) {
 
         //Initialize Components
         imageView = findViewById(R.id.imageView);
-        statusView = findViewById(R.id.textView_status);
+        amount = findViewById(R.id.amount);
+        upiid = findViewById(R.id.UpiId);
+        note = findViewById(R.id.Note);
+        payee = findViewById(R.id.PayeeName);
```

Comment:
Variable names should follow camel case format. Like `etAmount`, `etUpiId`, etc.